torchsde/_core/methods/milstein.py

```diff
@@ -12,30 +12,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ...settings import SDE_TYPES, NOISE_TYPES, LEVY_AREA_APPROXIMATIONS
+import abc
+
```

Comment:
Tiniest nitpick: no new line here between 'abc' and 'math'. (The organisation on the imports is grouped by standard library, then external libraries, then local files n levels above, then local files n-1 levels above, etc.)

Reply:
Sure, just pushed it.